### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Recommendation for renderView()

## Subject

Recommendation for renderView()

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,17 @@ $dompdf->streamHtml($html, "document.pdf");
 $dompdf->getPdf($html);
 ```
 
+If you use Twig to create the content, make sure to use `renderView()` instead of `render()`.
```
Comment (core23) [inline README.md]: This is not related to twig. `Twig\Environment` has no `renderView` method

Comment (ThomasLandauer) [reply README.md]: Sorry, dont't understand :-(
With `$html = $this->render()` I got those headers. So what's your preferred method?

Comment (core23) [reply README.md]: I way thinking about this class https://github.com/twigphp/Twig/blob/2.x/lib/Twig/Environment.php#L287 which has no `renderView` method.

Can you please add a short note, that you are talking about the controller (https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php#L201) which has this method.
